mkdocs.yml

```diff
@@ -1,6 +1,6 @@
 site_name: django-user-management
 pages:
-- [index.md, Home]
+- [README.md, Home]
```

Comment:
Hm, I'm not sure. I only noticed that `index.md` was used here so I renamed it. Perhaps `README.md` could be a symlink to `index.md`, thus preserving the `mkdocs` functionality?